index.html

```diff
@@ -33,224 +33,252 @@
-        </ul>
-        <button type="button" class="btn">See Project</button>
+      <div class="work-info4">
+        <div class="image-container">
+          <img src="./images/work-img4.png" alt="multi" class="work-img" />
+        </div>
+        <div class="content1">
+          <h3 class="work-title">
+            Multi-Post <br />
+            Stories
+          </h3>
+          <h4>
+            CANOPY <span class="dot tag"> Back End dev </span>
+            <span class="dot tag">2015</span>
+          </h4>
+          <p class="title-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="stack-group">
+            <li class="stack">html</li>
+            <li class="stack">css</li>
+            <li class="stack">javascript</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
+        </div>
       </div>
     </section>
     <!-- About Section -->
     <section class="about-me">
-      <h1 class="headline-main">About <br />Myself</h1>
-      <p class="headline-paragraph">
-        I’m a software developer! I can help you build a product , feature or
-        website. Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <h3 class="headline-connect">LET'S CONNECT</h3>
+      <div class="content2">
+        <h1 class="headline-main">About <br />Myself</h1>
+        <p class="headline-paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3 class="headline-connect">LET'S CONNECT</h3>
+
+        <ul class="headline-icons">
+          <li>
+            <a href="">
+              <i class="fa-xl"
+                ><img src="./images/icon-twitter.svg" alt="twitter"
+              /></i>
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa-xl"
+                ><img src="./images/icon-1.svg" alt="linkedin"
+              /></i>
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa-xl"
+                ><img src="./images/icon-2.svg" alt="medium"
+              /></i>
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa-xl"
+                ><img src="./images/icon-4.svg" alt="github"
+              /></i>
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa-xl"><img src="./images/icon-3.svg" alt="m" /></i>
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="btn-resume">Get my resume</button>
+      </div>
```

Comment:
- [ ] I noticed here the about and languages sections are not aligned the same as the Figma design please align them to match the Figma design you can use flex to align them💪🏻💪🏻and Kindly add some margin to match the design because I can see it is very close to the Form section 

- [ ] Kindly put the same icon for the javascript as the design you can import it from the Figma 🔥🔥

- your design
![image](https://user-images.githubusercontent.com/58553711/177900804-991d46e9-305c-405f-9b36-4097a709cb44.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177901019-8b6325a0-d889-46ac-9c43-e8fcc95b3794.png)

Reply:
Correction effected.
Thanks for the review.